core/services/relay/evm/mercury/transmitter.go

```diff
@@ -243,6 +259,29 @@ func (mt *mercuryTransmitter) runloop() {
 				mt.lggr.Errorw("Transmit report failed; mercury server returned error", "unpackErr", unpackErr, "validFromBlock", validFrom, "currentBlock", currentBlock, "req", t.Req, "response", res, "reportCtx", t.ReportCtx, "err", res.Error, "code", res.Code)
 			}
 		}
+
+		if err := mt.orm.DeleteTransmitRequest(runloopCtx, t.Req); err != nil {
+			mt.lggr.Errorw("Failed to delete transmit request record", "error", err, "req", t.Req, "reportCtx", t.ReportCtx)
+			return
+		}
+	}
+}
+
+func (mt *mercuryTransmitter) runPruneLoop() {
+	runloopCtx, cancel := mt.stopCh.Ctx(context.Background())
+	defer cancel()
+	ticker := time.NewTicker(PruneFrequency)
```

Comment:
You may wish to consider using a `time.After(utils.WithJitter(PruneFrequency))` to prevent the stampeding herd problem.